Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM php:8-apache  
+RUN set -ex && bash -c "set -euxo pipefail && install <(curl -fsSLo - "$(curl -sLI -o /dev/null -w '%{url_effective}' "https://github.com/tianon/gosu/releases/latest" | sed 's/tag/download/')/gosu-amd64") /usr/local/bin/gosu" && gosu --version
+EXPOSE 80
+ENTRYPOINT [ "docker-entrypoint.sh" ]
+
+COPY docker-entrypoint.sh /usr/local/bin/
+
+RUN set -ex && docker-php-ext-install pdo_mysql
+COPY --chown=www-data:www-data ./src/ /var/www/html/
```

Comment:
When testing the container I noticed that the php errors where not catched. Please at something like:
```
RUN { \
    touch /usr/local/etc/php/php.ini; \
    echo "log_errors = On"; \
    echo "error_log = /dev/stderr" \
    echo "error_reporting = E_ALL"; \
} > /usr/local/etc/php/php.ini
```

Reply:
Also, nice catch! I was wondering why I wasn't seeing any errors! :sweat_smile: 

I actually noticed that the docker container has a builtin "production" php.ini version. I tried that and it enabled the php logging.

I was curious is there a reason why I shouldn't use it? Here is version of the file without any comments ( in case you know what they all are and if they are good or not ):
[php.ini-production.txt](https://github.com/hashtopolis/server/files/7148202/php.ini-production.txt)

Also, here is the commented one for context: 
[php.ini-production-comments.txt](https://github.com/hashtopolis/server/files/7148206/php.ini-production-comments.txt)

it looks like it has all the options you had specified in the command you posted.

Reply:
I saw the production.ini as well. To be honest I don't know if you should use it, I looked around for some documentation on the php container and it was a bit limited... I guess if it is called production it should be fine right? :satisfied:

Reply:
> I guess if it is called production it should be fine right? :laughing: 

lol, this :arrow_up: is exactly what I was thinking :grin: , but it is up to you :slightly_smiling_face:

Reply:
I think this should work, just open up the conversation again if we should change this :grin: